src/storage/src/source/mysql.rs

```diff
@@ -223,6 +216,15 @@ impl SourceRender for MySqlSourceConnection {
     }
 }
 
+#[derive(Clone, Debug)]
+struct SubsourceInfo {
+    output_index: usize,
+    name: MySqlTableName,
+    desc: MySqlTableDesc,
+    initial_gtid_set: String,
```

Comment:
This probably needs to be an `Antichain<GtidPartition>` too, right?

Reply:
Yes it just depends on where we want to convert the string to the antichain representation, but I'll move it here for cleanliness.